scripts/hallucinate_fragalysis.py

```diff
@@ -369,6 +372,7 @@ def mp_func(
         dimer,
         f"{out_base}/docked.sdf",
         rmsd,
+        # TODO: If `dock_sys` is not "posit", then `posit_prob` might be undefined here.
```

Comment:
```suggestion
```
if `dock_sys` is `"hybrid"` then `posit_prob` is manually set to `-1` on line 295